CRM/Civicase/Service/CaseMenu/AllRoute.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Class for All route.
+ *
+ * Used for listing all the Cases for the associated Category Type Category.
+ */
+class CRM_Civicase_Service_CaseMenu_AllRoute extends CRM_Civicase_Service_CaseMenu_Route {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function label() {
```

Comment:
Function name should have get prefix. Same for other functions here.
Like getLabel, getName etc.

Reply:
I updated the description of the PR, since I omitted to put an explanation about the route class and the related methods. Basically, getLabel (and getName, etc) will be handled by the PHP magic methods https://www.php.net/manual/en/language.oop5.magic.php
Let me know if it is not clear enough.